### ISSUE TITLE
Improve the icetray-start+tarball documentation (Trac #2022)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2014]
Hi,

it would be nice if someone could update the icetray-start documentation at http://software.icecube.wisc.edu/documentation/info/cvmfs.html#self-contained-icetray-scripts

The documentation says that one has to supply "a tarball URL" but never specifies what this means.
I assumed this would just mean a filepath to the tarfile which is apparently wrong.
It would be nice if this was a bit more clear on how to run for example your own tarball.

Thanks!

Migrated from https://code.icecube.wisc.edu/ticket/2022
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:13", 
    "description": "Hi,\n\nit would be nice if someone could update the icetray-start documentation at http://software.icecube.wisc.edu/documentation/info/cvmfs.html#self-contained-icetray-scripts\n\nThe documentation says that one has to supply \"a tarball URL\" but never specifies what this means.\nI assumed this would just mean a filepath to the tarfile which is apparently wrong.\nIt would be nice if this was a bit more clear on how to run for example your own tarball.\n\nThanks!", 
    "reporter": "flauber", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067313248429", 
    "component": "cvmfs", 
    "summary": "Improve the icetray-start+tarball documentation", 
    "priority": "trivial", 
    "keywords": "tarball, icetray-start, icetray", 
    "time": "2017-05-15T11:16:56", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170628T17:13:29: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20170628T17:14:59: david.schultz changed owner from "david.schultz" to "jvansanten"

**Comment by icecube-trac** [comment]
Trac update at 20170818T16:02:15: david.schultz changed milestone from "" to "September 2017"

**Comment by icecube-trac** [comment]
Trac update at 20180607T19:31:48: olivas changed milestone from "September 2017" to "Summer Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20180612T07:19:16: 
* jvansanten changed status from "assigned" to "closed"
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 3065:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="3065"
> Describe naming convention for tarballs used in icetray-start.
> 
> Finally closes #2014.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:45:31: nwhitehorn commented:

> In changeset 170022:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170022"
> Describe naming convention for tarballs used in icetray-start.
> 
> Finally closes #2014.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:13: 
* olivas commented:

> Milestone Summer Solstice 2018 deleted


* olivas changed milestone from "Summer Solstice 2018" to ""
